### PR TITLE
update intros to be shorter and easier to read

### PR DESCRIPTION
**Pull request by codehag**
I did a through read through of the intros, and realized that we should be using fewer words and shorter sentences. It can be hard to follow as it is. Let me know what you folks think

### CODE REVIEW

**Comment by codehag** [comment]
spellling is not my strong suite.

File: _includes/intro.html [hunk 1]
```diff
@@ -32,11 +33,17 @@ <h2 class="section-headline">Contribute</h2>
       <h2 class="section-headline">Specs</h2>
       <p>
         We develop the JavaScript (formally, ECMAScript) specification
-        <a href="https://github.com/tc39/ecma262/">on GitHub</a> and at meetings every two months. We follow a
-        <a href="https://tc39.github.io/process-document/">four stage process</a> for new
-        <a href="https://github.com/tc39/proposals/">language feature proposals</a>. See our
-        <a href="https://github.com/tc39/agendas/">meeting agendas</a> and
-        <a href="https://github.com/tc39/tc39-notes/">minutes</a> to learn more.
+        <a href="https://github.com/tc39/ecma262/">on GitHub</a>. We meet every two
+        months. To learn more about the process, please take a look at the
```
Comment (rkirsling) [inline _includes/intro.html]: I feel like splitting off `We meet every two months.` as its own sentence makes it less clear what we're meeting about. Would `and meet to discuss proposals every two months` work?

File: _includes/intro.html [hunk 2]
```diff
@@ -16,9 +16,10 @@ <h2 class="section-headline">TC39</h2>
     <div class="g-col-4">
       <h2 class="section-headline">Contribute</h2>
       <p>
-        TC39 welcomes contributions from the JavaScript community, whether it is feedback on existing proposals, improved documentation,
-        testing, implementations, or even language feature ideas.
-        <!-- TODO: Make each of the things in the preceding sentence links to the appropriate guides -->
+       TC39 welcomes contributions. This includes proposal feedback, documentation, writing tests or
+       implementations, and language feature ideas.
```
Comment (rkirsling) [inline _includes/intro.html]: `This includes` sounds kind of like a complete list to me. If we're dropping `the JavaScript community` (_which is probably good in the interest of avoiding "us vs. them"_), maybe we could direct it right at the reader, like `You can help by...`?

File: _includes/intro.html [hunk 3]
```diff
@@ -16,9 +16,10 @@ <h2 class="section-headline">TC39</h2>
     <div class="g-col-4">
       <h2 class="section-headline">Contribute</h2>
       <p>
-        TC39 welcomes contributions from the JavaScript community, whether it is feedback on existing proposals, improved documentation,
-        testing, implementations, or even language feature ideas.
-        <!-- TODO: Make each of the things in the preceding sentence links to the appropriate guides -->
+       TC39 welcomes contributions. You can help by giving proposals feedback, improving documentation, writing tests or
```
Comment (rkirsling) [inline _includes/intro.html]:
```suggestion
       TC39 welcomes contributions. You can help by giving feedback on proposals, improving documentation, writing tests or
```
(Otherwise `giving proposal feedback` could also work.)

File: _includes/intro.html [hunk 4]
```diff
@@ -16,9 +16,10 @@ <h2 class="section-headline">TC39</h2>
...
+       implementations, and suggesting language feature ideas.
```
Comment (rkirsling) [inline _includes/intro.html]: Maybe still better to keep this as an `or`? Up to you though.
```suggestion
       implementations, or suggesting language feature ideas.
```

File: _includes/intro.html [hunk 5]
```diff
@@ -32,11 +33,17 @@ <h2 class="section-headline">Contribute</h2>
       <h2 class="section-headline">Specs</h2>
       <p>
         We develop the JavaScript (formally, ECMAScript) specification
-        <a href="https://github.com/tc39/ecma262/">on GitHub</a> and at meetings every two months. We follow a
-        <a href="https://tc39.github.io/process-document/">four stage process</a> for new
-        <a href="https://github.com/tc39/proposals/">language feature proposals</a>. See our
-        <a href="https://github.com/tc39/agendas/">meeting agendas</a> and
-        <a href="https://github.com/tc39/tc39-notes/">minutes</a> to learn more.
+        <a href="https://github.com/tc39/ecma262/">on GitHub</a> and meet every two
+        months to discuss the proposals. To learn more about the process, please take a look at the
```
Comment (rkirsling) [inline _includes/intro.html]: Feel free to disagree, but the `the` here made me reflexively ask, "which ones exactly?"
```suggestion
        months to discuss proposals. To learn more about the process, please take a look at the
```

**Review by rkirsling — APPROVED** [review]
